net/upnp/src/etc/inc/plugins.inc.d/miniupnpd.inc

```diff
@@ -167,7 +171,7 @@ function miniupnpd_configure_do($verbose = false)
 
         if (!empty($ifaces_active)) {
             /* override wan ip address, common for carp, etc */
-            if (!empty($upnp_config['overridewanip'])) {
+            if (!empty($upnp_config['overridewanip']) && empty($upnp_config['stun_host'])) {
```

Comment:
if this change is needed the logic should continue with `stun_host` configuration instead of skipping unrelated lines to below.